src/app/commons/TextInputForm/TextInputForm.js

```diff
@@ -23,7 +23,7 @@ const TextInputForm = ({placeholder, onSubmit}) => {
     return (
         <div className="text-input-form">
             <form className="tag-form" onSubmit={handleSubmit}>
-                <div className="field has-addons">
+                <div className="field has-addons" style={{ paddingLeft: "0rem" }}>
```

Comment:
add className `pl-0` (see: https://bulma.io/documentation/helpers/spacing-helpers/) 
instead of `style={{ paddingLeft: "0rem" }}`

Reply:
Thank u!